docs/docs/debug/argon2.md

```diff
@@ -0,0 +1,22 @@
+---
+id: argon2
```

Comment:
Rename this to `performance-out-of-memory-password-hashing-argon2` for the extra SEO juice